tests/bdd/features/snapshot/list/list.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Volume Snapshot listing
+
+  Background:
+    Given a deployer cluster
+
+  # LISTING BY DIFFERENT CRITERIAS
+  Scenario Outline: Volume snapshots list
+    Given we have a single replica <publish_status> volume
+    And we have created more than one snapshot for the volume
+    Then the snapshots should be listed by <filter>
```

Comment:
Would be worth it adding  another bdd similar to this, but after we have disruption, like node killed and brought back or restarted ? cc @tiagolobocastro

Reply:
I think we are doing that kind of validation in create bdd tests.

Reply:
We are not right? We only use the `get by snapshot id` call in all other tests.

Reply:
Yes I was referring to that get by snapshot id after a node disruption. Is it worth running all filters for listing after a node reboot? The existence of snapshot is to be established primarily.

Reply:
Yeah might be worth it to have one scenario where we stop/kill a node and list, and then we bring it back and list. 
Just to be on the safe side all is as expected and no conversion missing.

Reply:
I think we also should add here checks for all snapshot fields, example:
```
    And the snapshot status ready to clone should be true
    And the snapshot source size should be equal to the volume size
    And the snapshot size should be equal to the volume size
    And the snapshot timestamp should be in the RFC 3339 format
    And the snapshot timestamp should be within 1 minute of creation (UTC)
    And the volume snapshot state has a single online replica snapshot
    And the replica snapshot's timestamp should be in the RFC 3339 format
    And the replica snapshot's timestamp should be within 1 minute of creation (UTC)
```

Reply:
Shall we rather have a single line in bdd saying:
`And the snapshot fields must be valid`

Then in the test implementation, keep an array or dictionary of all the fields that are to be validated, and validate them. This would avoid changing bdd if any new field is added/removed in future.

Reply:
One can have that if we have multiple scenarios but we should have one scenario with that explicitly so it's clear. However  I've added that on created, so maybe that's enough to keep it explicit there and simple here CC @Abhinandan-Purkait